src/pages/logout.tsx

```diff
@@ -21,6 +21,8 @@ function Logout() {
     };
     userLogout();
     localStorage.removeItem("token");
+    localStorage.removeItem("preferredLanguage");
```

Comment:
@AkshataKatwal16 dont remove the preferred language on logout

Reply:
okay...as per sanket suggestion i have removed that,now reverted back that line